src/styles/app.css

```diff
@@ -128,3 +128,23 @@ h5,
 h6 {
     font-family: "Noticia Text", serif;
 }
+
+div .hUgJMD .remirror-editor.ProseMirror {
```

Comment:
issue: changes in the app.css are an anti-pattern, use styled-components or a file for the specific component